spec/Section 5 -- Validation.md

```diff
@@ -1419,6 +1432,7 @@ For example the following document will not pass validation.
     * Let {fieldName} be the name of {fieldDefinition}.
     * Let {field} be the input field in {fields} named {fieldName}
     * {field} must exist.
+    * {field} must not be deprecated.
```

Comment:
@fotoetienne It's the incorrect place for this validation.
"Section 5" validates queries, we don't want to fail query because field marked as deprecated in the schema.
This validation should be done during schema validation, please move it here:
https://spec.graphql.org/draft/#sel-IAHhBZHBAABABP9xB

Reply:
Good catch, thanks!